pkg/controller/admissionchecks/provisioning/admissioncheck_reconciler.go

```diff
@@ -61,6 +64,7 @@ func (a *acReconciler) Reconcile(ctx context.Context, req reconcile.Request) (re
 
 	if currentCondition.Status != newCondition.Status {
 		apimeta.SetStatusCondition(&ac.Status.Conditions, newCondition)
+		a.record.Eventf(ac, corev1.EventTypeNormal, "UpdatedAdmissionChecks", "Admission checks status is changed to %s", newCondition.Status)
 		return reconcile.Result{}, a.client.Status().Update(ctx, ac)
```

Comment:
Record the event only if the status update is successful.

Reply:
+1